### DISCUSSION TITLE
Embedded Loom videos not downloadable

### DISCUSSION THREAD

**Discussion by ElMeik** [aclap-dev/video-downloadhelper#1376] [Help and Q&A]
### Your question

Team - i see some old requests and that his has been created - but - do we have a solution in place since I still encounter issues DLH not being able to download embedded Loom Videos e.g. from Skool? 

Best regards

### Addon & CoApp version and details

V 9.1.1.0 Chrome Extension

**Comment by Mattmjs** [comment]
They're aware of the problem and there is a planned fix, when that fix is coming I don't know. How much time do you have on the
loom videos ?

**Reply by ElMeik** [reply]
Thx for the update Mattmjs :-)

With "how much time" you mean the length of the videos? Well there are couple of vids with different length..none of them is possible to being downloaded :-(

But good to know that the team is working on this issue :-D

*cheers*

**Comment by Mattmjs** [comment]
I was asking if you have a limited time before you lose access to the
videos.
If not you could probably wait for when there is fix.

**Reply by ElMeik** [reply]
ah ok..understand - no no - I got no limit in that way...thx Mattmjs...it would just be genius if this great app would be able to handle THOSE videos as well...really love it !

**Reply by Mattmjs** [reply]
Maybe there is another or maybe there isn't, I wouldn't know unless you recorded the network activity.
With the developer tools open and selecting the Network tab, reload the page and once it stops click the button to save it.
Then you add it here as a .zip file. See the image.

![chrome dev tools](https://github.com/user-attachments/assets/7e3bd072-2bc3-4d5e-83d1-3904d4634925)
